2020-11-24 16:30:57 :: ERROR :: FileName : new.py, LineNumber : 725, ExceptionType : <class 'NameError'>, Exception : name 'synapseQueriesFilePath' is not defined

